.github/workflows/amd-ci.yml

```diff
@@ -53,12 +52,40 @@ jobs:
       with:
         python-version: '3.10'
 
-    - name: Install dependencies
+    - name: Check Docker Version
+      run: docker version
+
+    - name: Check Ubuntu version
+      run: lsb_release -a
+
+    - name: Check Hardware Specs
+      run: lscpu
+    
+    - name: ROCM-SMI Output
       run: |
-        python -m pip install --upgrade pip
-        pip install -e ".[dev]"
+        rocm-smi
+        rocm-smi --showproductname
 
-    - name: Run tests
+    - name: Setup Dependencies
+      run: |
+        cp -r /opt/rocm/share/amd_smi ./
+        cd amd_smi
+        python -m pip install -e .
+        cd ..
+        python -m pip install pytest pytest-xdist pytest-rerunfailures pytest-flakefinder pytest-cpp
+        python -m pip uninstall -y torch torchvision
+        python -m pip install --pre \
```

Comment:
this is a bit nasty. Can we have a easy way to install amd dep? Like `pip install liger-kernel[amd]`

Reply:
Addressing this in PR https://github.com/linkedin/Liger-Kernel/pull/436.